core/src/main/java/org/axonframework/eventsourcing/eventstore/EventStorageEngine.java

```diff
@@ -116,4 +119,19 @@ default DomainEventStream readEvents(String aggregateIdentifier) {
      * @return An optional with a snapshot of the aggregate
      */
     Optional<DomainEventMessage<?>> readSnapshot(String aggregateIdentifier);
+
+    /**
+     * Returns the last known sequence number for the given {@code aggregateIdentifier}.
+     * <p>
+     * While it's recommended to use the sequence numbers from the {@link DomainEventStream}, there are cases where
+     * knowing the sequence number is required, without having read the actual events. In such case, this method is a
+     * viable alternative.
+     *
+     * @param aggregateIdentifier The identifier to find the last sequence number for
```

Comment:
Nit: Shouldn't there be dots at the end of the `@param` and `@return` descriptions?

Reply:
No, a dot is mandatory in the first line of javadoc, to demarcate the first sentence (which is used as summary). As far as I know, it's not mandagory on params and returns.